talk/app/talk-cli.hoon

```diff
@@ -857,15 +853,9 @@
     ::  +act: build action card
     ::
     ++  act
-      |=  [what=term app=term =cage]
+      |=  [=path app=term =cage]
       ^-  card
-      :*  %pass
-          /cli-command/[what]
-          %agent
-          [our-self app]
-          %poke
-          cage
-      ==
+      [%pass path %agent [our-self app] %poke cage]
```

Comment:
This is a `wire`, not a `path`. (Their types have the same shape, but the semantics are different. Wires are internal-only references, paths are more akin to API identifiers.)

Same in `+flee`.